### PR TITLE
Add tr translation to SAO Utils 2

### PR DESCRIPTION
**Pull request by Lawliet0**
Thank you for creating great software, this is a Turkish localization, I've already tested it on the Steam version of SAO Utils 2 and I'm happy with the translation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
-# locales
```
Comment (Lawliet0) [inline README.md:1]: Turkish

**Comment by Lawliet0** [comment]
How do I update the README.md

**Comment by GPBeta** [comment]
Hi, I think you are editing on a wrong branch.
Your previous translation works are based on the Beta/Steam branch (https://github.com/NERvGear/locales/tree/Steam).
While this Progressive branch is used for translating SAO Utils 2.

**Comment by Lawliet0** [comment]
I translated it in sao utils 2, I tested it on my own pc.
But I forgot how to upload it to github.
sdfsdf

**Comment by GPBeta** [comment]
Oh, that's great. You will need to pull the latest version of Progressive branch to your local repo, commit the new translations and create a new pull request.
If you don't know how to do, you can zip and upload your translations here. I'll commit them for you :)

**Pull request closed without merging**